components/camel-google-pubsub/src/main/java/org/apache/camel/component/google/pubsub/GooglePubsubConstants.java

```diff
@@ -19,16 +19,12 @@
 public final class GooglePubsubConstants {
 
     public static final String MESSAGE_ID = "CamelGooglePubsub.MessageId";
-    public static final String ACK_ID = "CamelGooglePubsub.MsgAckId";
     public static final String PUBLISH_TIME = "CamelGooglePubsub.PublishTime";
     public static final String ATTRIBUTES = "CamelGooglePubsub.Attributes";
-    public static final String ACK_DEADLINE = "CamelGooglePubsub.AckDeadline";
 
     public enum AckMode {
         AUTO, NONE
     }
 
-    private GooglePubsubConstants() {
```

Comment:
Any reason to remove the private constructor since it was meant to be a singleton class?

Reply:
Right, I've introduced it again

Reply:
Failing tests were due to missing @BindToRegistry("prop")